src/components/Controllers/Files/Files.tsx

```diff
@@ -10,6 +10,7 @@ export interface FilesProps {
   client: Client
   initialUpload?: boolean
   initialDataPackage?: boolean
+  fileItemAdded?: boolean
```

Comment:
Not sure I understand where you use it

Reply:
It is to To trigger listFiles() [here](https://github.com/frictionlessdata/application/blob/main/src/components/Controllers/Files/Layout.tsx#L15) after export api is called.

[export.webm](https://user-images.githubusercontent.com/5582809/223500198-ef3ee6ab-f5c4-4530-a486-822ec4b5b170.webm)

Not sure if this is the right way.